crates/header-translator/src/rust_type.rs

```diff
@@ -1672,22 +1672,22 @@ impl fmt::Display for Ty {
                 Inner::Id {
                     ty:
                         ty @ IdType::Class {
-                            id,
                             params: TypeParams::Empty,
                             ownership: None,
+                            ..
                         },
                     is_const: _,
                     lifetime: _,
                     nullability: Nullability::Nullable | Nullability::Unspecified,
-                } if id.is_nsstring() || id.is_nsdata() => {
+                } => {
```

Comment:
I suspect I'm going to go back on this a bit at some point (or, at least special-case `NSString`, since it would be nice to make safe), but for now it's fine